src/tokens/colors.json

```diff
@@ -10,7 +10,8 @@
     },
     "ui-details": {
       "midnight-green": "#002219",
-      "digital-moss": "#0c4d33"
+      "digital-moss": "#0c4d33",
+      "digital-dark-grass": "#1A8924"

```

Comment:
shouldn't we also have `"digital-grass": "#1A8924"` as a fallback? i am sure there are people using `$ys-color-digital-grass` otherwise it sounds like a BREAKING CHANGE.

Reply:
nvm i missunderstood the task 👍